.github/workflows/cherry-pick-pr.yml

```
@@ -0,0 +1,24 @@
+name: Cherry-pick PRs from next to master
+
+on:
+  pull_request:
+    branches:
+      - next
+    types: ['closed']
+
```

We need to remove all permission per default: https://github.com/mui/mui-x/security/code-scanning/18

```suggestion
permissions: {}
```